app/models/user.rb

```diff
@@ -13,4 +13,14 @@ class User < ApplicationRecord
   validates :email, uniqueness: true, presence: true
   validates :password_digest, presence: true
   validates :username, uniqueness: true, presence: true
+
+  def update_target(language_id, user)
+    user_language = user.user_languages.find_by(fluency_level: 'target')
+    user_language.update(language_id: language_id) if user_language
+  end
+
+  def update_native(language_id, user)
+    user_language = user.user_languages.find_by(fluency_level: 'native')
+    user_language.update(language_id: language_id) if user_language
+  end
```

Comment:
@lkriffell Do you get an error if you call line 24 and they don't have a user_language? Wondering if we can just add it if they don't have an existing one vs. sending blank.

Reply:
It might, i'll check

Reply:
I can go ahead and merge and maybe we can revisit in the future! @lkriffell